Gems/Atom/RPI/Code/Include/Atom/RPI.Reflect/Image/StreamingImageAsset.h

```diff
@@ -85,6 +85,19 @@ namespace AZ
             //! Get image data for specified mip and slice. It may return empty array if its mipchain assets are not loaded
             AZStd::array_view<uint8_t> GetSubImageData(uint32_t mip, uint32_t slice);
 
+            //! Get single image pixel value for specified mip and slice
+            template<typename T>
+            T GetSubImagePixelValue(uint32_t x, uint32_t y, uint32_t componentIndex = 0, uint32_t mip = 0, uint32_t slice = 0);
+
+            //! Retrieve a region of image pixel values (float) for specified mip and slice
+            void GetSubImagePixelValues(AZStd::pair<uint32_t, uint32_t> topLeft, AZStd::pair<uint32_t, uint32_t> bottomRight, AZStd::array_view<float> outValues, uint32_t componentIndex = 0, uint32_t mip = 0, uint32_t slice = 0);
```

Comment:
`AZStd::array_view<float>` should be switched to the recently added `AZStd::span<float>`

Reply:
Will update this one and the others, thanks!